app/gateway/performance/performance-benchmarks.md

```diff
@@ -12,13 +12,116 @@ works_on:
+| Rate limit and basic auth   | 100 Routes, 100 Consumers   | 89378.4                   | 10.20    | 5.37     |
+{% endif_version %}
+
+
+### Test environment
+
+Kong ran these tests in AWS using EC2 machines. We used [Kubernetes taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) to ensure that {{site.base_gateway}} is on its own node while the load testing and observability tools are on their own separate nodes in the same cluster.
+
+{{site.base_gateway}} ran on a single dedicated instance of c5.4xlarge, and the two nodes for the observability stack and K6 ran on dedicated c5.metal instances. We used the metal instances for the observability load generation toolchain to ensure they aren't resource-constrained in any way. Since [K6 is very resource-demanding](https://k6.io/docs/testing-guides/running-large-tests/#hardware-considerations) when generating a high amount of traffic during tests, we observed that using smaller or less powerful instances for the toolchain caused the observability load generation tools to be a bottleneck for {{site.base_gateway}} performance.
+
+### Test configuration
+
+For these tests, we changed the number of worker processes to match the number of available cores to the node running {{site.base_gateway}}, which was 16 vCPU. Accordingly, we set the number of processes to 16. This follows [Kong’s overall performance guidance](/gateway/performance/resource-sizing-guidelines/). Outside of this change, no other tuning was made.
+
+## Conduct your own performance test using Kong's test suite
+
+You can use [Kong's public test suite repo](https://github.com/Kong/kong-gateway-performance-benchmark/tree/main) to help you spin up an EKS cluster with {{site.base_gateway}}, Redis, Prometheus, and Grafana installed. Additionally, it will configure [K6](https://k6.io/), a popular open source load testing tool. You can use this test suite to conduct your own performance tests.
+
+Once the cluster is generated, you can apply the [provided YAML](https://github.com/Kong/kong-gateway-performance-benchmark/tree/main/deploy-k8s-resources/kong_helm) to configure {{site.base_gateway}} for the included test cases, and the observability plugins for metrics scraping by the Prometheus instance already provisioned in the cluster. If you’d rather define your own test scenarios, you can also define the {{site.base_gateway}} configuration you want to test and apply it to the cluster.
 
-pull content from https://docs.konghq.com/gateway/latest/production/performance/performance-testing/
\ No newline at end of file
+From there, you can use the [included bash scripts to run K6 tests](https://github.com/Kong/kong-gateway-performance-benchmark/tree/main/deploy-k8s-resources/k6_tests). After the tests complete, you can `port-forward` into the cluster and view the Grafana dashboard with the performance results.
```

Comment:
```suggestion
From there, you can use the [included bash scripts to run K6 tests](https://github.com/Kong/kong-gateway-performance-benchmark/tree/main/deploy-k8s-resources/k6_tests). After the tests complete, you can `port-forward` into the cluster and view the Grafana dashboard with the performance results.

For more information about how to conduct your own performance benchmark of {{site.base_gateway}}, see [Establish a {{site.base_gateway}} performance benchmark](/gateway/performance/establish-performance-benchmark).
```
I know we already link to this in the info box, but I wanted to add it here since it seemed relevant.